app/src/interfaces/translations/translations.vue

```diff
@@ -217,234 +284,199 @@ export default defineComponent({
-					emit(
-						'input',
-						props.value.map((val) => {
-							if (typeof val === 'string' || typeof val === 'number') return val;
+				if (pkField in edits === false) {
+					const newIndex = copyValue.findIndex((item) => typeof item === 'object' && item[langField] === lang);
 
-							if (val[translationsLanguageField.value!] === editedLanguage) {
+					if (newIndex !== -1) {
+						if (Object.keys(edits).length === 1 && langField in edits) {
+							copyValue.splice(newIndex, 1);
+						} else {
+							copyValue[newIndex] = edits;
+						}
+					} else {
+						copyValue.push({
+							...edits,
+							[langField]: lang,
+						});
+					}
+				} else {
+					const initialValues = items.value.find((item) => item[langField] === lang);
+
+					copyValue = copyValue.map((item) => {
+						if (typeof item === 'number' || typeof item === 'string') {
+							if (edits[pkField] === item) {
 								return edits;
+							} else {
+								return item;
 							}
-
-							return val;
-						})
-					);
-				} else {
-					if (editing.value === '+') {
-						emit('input', [...(props.value || []), edits]);
-					} else {
-						emit(
-							'input',
-							props.value.map((val) => {
-								if (typeof val === 'string' || typeof val === 'number') {
-									if (val === editing.value) return edits;
+						} else {
+							if (edits[pkField] === item[pkField]) {
+								if (isEqual(initialValues, { ...initialValues, ...edits })) {
+									return edits[pkField];
 								} else {
-									if (val[pkField] === editing.value) return edits;
+									return edits;
 								}
-
-								return val;
-							})
-						);
-					}
+							} else {
+								return item;
+							}
+						}
+					});
 				}
 
-				editing.value = false;
+				emit('input', copyValue);
 			}
 
-			function cancelEdit() {
-				edits.value = {};
-				editing.value = false;
-			}
+			return { items, firstItem, updateValue, secondItem, firstItemInitial, secondItemInitial };
 		}
+	},
+});
+</script>
+<style lang="scss" scoped>
+.divider {
+	display: flex;
+	gap: 20px;
 
-		function usePreview() {
-			const loading = ref(false);
-			const error = ref(null);
-			const previewItems = ref<Record<string, any>[]>([]);
-
-			const { info: translationsCollectionInfo } = useCollection(translationsCollection);
-
-			const template = computed(() => {
-				if (!translationsPrimaryKeyField.value) return '';
+	.v-divider {
+		--v-divider-color: var(--primary-50);
 
-				return (
-					props.translationsTemplate ||
-					translationsCollectionInfo.value?.meta?.display_template ||
-					`{{ ${translationsPrimaryKeyField.value} }}`
-				);
-			});
+		margin-top: 40px;
 
-			watch(() => props.value, fetchPreviews, { immediate: true });
-			watch(languages, fetchPreviews, { immediate: true });
+		&.blue {
+			--v-divider-color: var(--blue-50);
+		}
+	}
+}
 
-			return { loading, error, previewItems, fetchPreviews, template };
+.translations {
+	display: flex;
+	gap: 24px;
```

Comment:
This gap doesn't match the gap between the dividers. We should also rely on form-horizontal-gap to make sure it's consistent with the rest of the half-width form on the page (as that size might change responsively)